scripts/build_docs.sh

```diff
@@ -3,7 +3,7 @@ set -e
 
 cp README.md docs/README.md
 # Alter the relative path of the README image for the docs.
-sed -i '1s/docs/./' docs/README.md
+gsed -i '1s/docs/./' docs/README.md
```

Comment:
What's the difference?

Reply:
I didn't mean to commit this, but on my mac I needed to use `gsed` because I got an error using `sed`.  This shouldn't be committed because it's system dependent and if you're on a Linux box then `sed` will be GNU `sed`.